Dockerfile

```diff
@@ -26,10 +26,11 @@ FROM registry.access.redhat.com/ubi9-minimal:9.4-1227
 
 WORKDIR /rhtap-cli
 
+COPY --from=quay.io/codeready-toolchain/oc-client-base:latest /usr/bin/kubectl /usr/bin/
```

Comment:
Please use `microdnf` to install `kubectl` instead.

Reply:
I couldn't find the package in `microdnf`.

Using
```
RUN curl --proto "=https" -L "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/${OC_VERSION}/openshift-client-linux.tar.gz" -o /tmp/openshift-client-linux.tar.gz && \
    tar --no-same-owner -xzf /tmp/openshift-client-linux.tar.gz && \
    mv oc kubectl /usr/local/bin && \
    oc version --client && \
    kubectl version --client
```
fails the build of non amd64 images.

Reply:
I'll leave it this way for the moment so we can release. We can try to do it better in a future release.